### PR TITLE
SSL_fix_format Fix wrong format used with SSL!

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
The two files that I am doing the pull do not have the proper format that we agreed when we write C code, so I am creating this PR to fix this.

##### Component Name
libnetdata/socket/security.c
libnetdata/socket/socket.c

##### Additional Information
PR #6212

### CODE REVIEW

File: libnetdata/socket/security.c [hunk 1]
```diff
@@ -30,13 +30,13 @@ void security_openssl_library()
```
Comment (cakrit) [inline libnetdata/socket/security.c]: Also remove the first and last space inside the `()` : e.g. `if ( !ctx ) -> if (!ctx)`

Comment (thiagoftsm) [reply libnetdata/socket/security.c]: The last pull fixed  the spaces as you requested!

File: libnetdata/socket/security.c [hunk 2]
```diff
@@ -12,8 +12,8 @@ int netdata_validate_server =  NETDATA_SSL_VALID_CERTIFICATE;
 
 static void security_info_callback(const SSL *ssl, int where, int ret) {
     (void)ssl;
-    if ( where & SSL_CB_ALERT ) {
-        debug(D_WEB_CLIENT,"SSL INFO CALLBACK %s %s",SSL_alert_type_string( ret ),SSL_alert_desc_string_long(ret));
+    if (where & SSL_CB_ALERT) {
+        debug(D_WEB_CLIENT,"SSL INFO CALLBACK %s %s",SSL_alert_type_string(ret),SSL_alert_desc_string_long(ret));
```
Comment (ilyam8) [inline libnetdata/socket/security.c]: spaces

File: libnetdata/socket/socket.c [hunk 3]
```diff
@@ -318,9 +318,9 @@ WEB_CLIENT_ACL socket_ssl_acl(char *ssl){
 
 WEB_CLIENT_ACL read_acl(char *st) {
     char *ssl = strchr(st,'^');
-    if (ssl){
+    if (ssl) {
         ssl++;
-        if ( !strncmp("SSL=",ssl,4)){
+        if ( !strncmp("SSL=",ssl,4)) {
```
Comment (ilyam8) [inline libnetdata/socket/socket.c]: leading space

File: libnetdata/socket/security.c [hunk 4]
```diff
@@ -116,10 +116,10 @@ static SSL_CTX * security_initialize_openssl_server(){
     return ctx;
 }
 
-void security_start_ssl(int type){
-    if ( !type){
+void security_start_ssl(int type) {
+    if (!type) {
         struct stat statbuf;
-        if ( (stat(security_key,&statbuf)) || (stat(security_cert,&statbuf)) ){
+        if ((stat(security_key,&statbuf)) || (stat(security_cert,&statbuf))) {
```
Comment (ilyam8) [inline libnetdata/socket/security.c]: looks like extra `()`

```c
(stat(security_key,&statbuf) || stat(security_cert,&statbuf))
```

File: libnetdata/socket/socket.c (hunk 3 above)
Comment (thiagoftsm) [reply libnetdata/socket/socket.c]: It is done with the last pull!

File: libnetdata/socket/security.c (hunk 4 above)
Comment (thiagoftsm) [reply libnetdata/socket/security.c]: I decided to use the extra () to isolate the results, I imagined that this was more simple to read, but I am fixing with the next pull.

File: libnetdata/socket/security.c (hunk 2 above)
Comment (thiagoftsm) [reply libnetdata/socket/security.c]: The last pull fixed this!

File: libnetdata/socket/security.c (hunk 4 above)
Comment (thiagoftsm) [reply libnetdata/socket/security.c]: The last pull fixed this!

File: libnetdata/socket/security.c [hunk 5]
```diff
@@ -207,7 +207,7 @@ int security_test_certificate(SSL *ssl){
     {
         char error[512];
         ERR_error_string_n(ERR_get_error(),error,sizeof(error));
```
Comment (ilyam8) [inline libnetdata/socket/security.c]: spaces

Comment (thiagoftsm) [reply libnetdata/socket/security.c]: This was fixed now!

**Pull request merged**